notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb

```diff
@@ -0,0 +1,354 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reading the timestamps and fluxes.\n",
+    "\n",
+    "Now that we have the target pixel file, let's store the timestamps and fluxes for use later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with fits.open(fits_file, mode=\"readonly\") as hdulist:\n",
+    "    tess_bjds = hdulist[1].data['TIME']\n",
+    "    raw_counts = hdulist[1].data['RAW_CNTS']\n",
+    "    calibrated_fluxes = hdulist[1].data['FLUX']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's examine one of these to see how Python stores the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(type(calibrated_fluxes))\n",
+    "print(calibrated_fluxes.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So these data are stored as a numpy array.  The first dimension is the number of cadences, followed by the size of the readout.  Thus, accessing calibrated_fluxes[4,:,:] should return to us the 11x11 grid of pixels from the **fifth** cadence (remember Python is zero-indexed, so the first cadence is stored in the zeroth index, the second in the first index, etc.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "calibrated_fluxes[4,:,:].shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Show the calibrated pixel values.\n",
+    "\n",
+    "Let's display the calibrated fluxes for one of the cadences in our target pixel file.  For now, we'll pick the fifth cadence (index number 4).  You could of course decide to look at multiple cadences by looping over each index in the first dimension of the structure (e.g., loop over calibrated_fluxes[i,:,:]).  You could even make an animated .gif of the fluxes and see how they change from cadence to cadence.  To get you started though, let's just display one of the cadence's flux values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start figure and axis.\n",
+    "fig, ax = plt.subplots()\n",
+    "# Display the calibrated fluxes as an image for the fifth cadence.\n",
+    "cax = ax.imshow(calibrated_fluxes[4,:,:], cmap=plt.cm.YlGnBu_r,\n",
+    "                origin=\"lower\")\n",
+    "# Let's define a title for the figure.\n",
+    "fig.suptitle(\"WASP-126 b Calibrated Fluxes - Sector 1, Fifth Cadence\")\n",
+    "# Add a color bar.\n",
+    "cbar = fig.colorbar(cax)\n",
+    "plt.show()"
+   ]
```

Comment:
Suggest adding newlines before each comment line to improve readability of the cell.